packages/@textlint/markdown-to-ast/test/fixtures/footnote-duplicate.text/output.json

```diff
@@ -24,30 +24,9 @@
                     "raw": "The NATO phonetic alphabet"
                 },
                 {
-                    "type": "LinkReference",
-                    "identifier": "^wiki",
-                    "referenceType": "shortcut",
-                    "children": [
-                        {
-                            "type": "Str",
-                            "value": "^wiki",
-                            "loc": {
-                                "start": {
-                                    "line": 1,
-                                    "column": 27
-                                },
-                                "end": {
-                                    "line": 1,
-                                    "column": 32
-                                }
-                            },
-                            "range": [
-                                27,
-                                32
-                            ],
-                            "raw": "^wiki"
-                        }
-                    ],
+                    "type": "footnoteReference",
```

Comment:
- [ ] Need to suppport `footnoteReference`

Previously, it is LinkReference.

Reply:
https://github.com/search?l=JavaScript&q=LinkReference+textlint&type=Code

Reply:
Probably, textlint-helper support new helper like `isMainContentStr(StrNode): boolean`

```js
        [Syntax.Str](node){
            if (helper.isChildNode(node, [Syntax.Link, "LinkReference", Syntax.Image, Syntax.BlockQuote, Syntax.Emphasis])) {
                return;
            }
```

is common pattern.